src/TLuaInterpreter.cpp

```diff
@@ -6575,22 +6575,25 @@ int TLuaInterpreter::exists(lua_State* L)
     QString name = getVerifiedString(L, __func__, 1, "name");
     QString type = getVerifiedString(L, __func__, 2, "type");
     Host& host = getHostFromLua(L);
-    int cnt = 0;
+    int count = 0;
     type = type.toLower();
-    if (type == "timer") {
-        cnt += host.getTimerUnit()->mLookupTable.count(name);
-    } else if (type == "trigger") {
-        cnt += host.getTriggerUnit()->mLookupTable.count(name);
-    } else if (type == "alias") {
-        cnt += host.getAliasUnit()->mLookupTable.count(name);
-    } else if (type == "keybind") {
-        cnt += host.getKeyUnit()->mLookupTable.count(name);
-    } else if (type == "button") {
-        cnt += host.getActionUnit()->findActionsByName(name).size();
-    } else if (type == "script") {
-        cnt += host.getScriptUnit()->findScriptId(name).size();
+    if (type == QStringLiteral("timer")) {
+        count += host.getTimerUnit()->mLookupTable.count(name);
```

Comment:
It is pointless using `+=` as the code structure is such that it is only ever going to do one value assignment to `cnt`/`count`!

:bulb: OTOH It would be reasonable if `type` was going to be an optional argument and its omission was going to report ALL items of *any* type with that name...

Reply:
Feedback applied. `type` would be a different discussion

Reply:
> Feedback applied. type would be a different discussion

Though, still valid within the title of this PR...